### PR TITLE
Make the standalone viewer downloadable 

### PR DESCRIPTION
**Pull request by ceriottm**
Travis deploys chemiscope_standalone.html when it builds the website. A button is added to make it downloadable.

### CODE REVIEW

**Comment by ceriottm** [comment]
This addresses partly #7, but not completely as it doesn't download the JSON

File: app/index.html [hunk 1]
```diff
@@ -180,6 +180,45 @@ <h5>References</h5>
...
+                    which is available on GitHub. 
+                </p>
+
+                <p>
+                    If you are a user, you may want to download a 
+                    <a href="chemiscope_standalone.html" download> 
+                    standalone version
+                    </a> of the standard viewer, which you can use to 
+                    display locally datasets prepared in the chemiscope 
+                    JSON format.
```
Comment (Luthaf) [inline app/index.html:215]: Should we add some basic instructions on how to use the standalone viewer (i.e. append the JSON file)? Or should we leave it as is and wait for the full implementation of "download current dataset as standalone"?

File: app/index.html [hunk 2]
```diff
@@ -180,6 +180,45 @@ <h5>References</h5>
...
+
+                <p>
+                    If you are a developer you may be interested in downloading
+                    the <a href="https://github.com/cosmo-epfl/chemiscope"> source code</a>,
+                    which is available on GitHub. 
+                </p>
+
+                <p>
+                    If you are a user, you may want to download a 
+                    <a href="chemiscope_standalone.html" download>
```
Comment (Luthaf) [inline app/index.html:211]: Regarding the "recursive" issue, we could have some CSS to hide this and a other part of the page (such as the list of examples) which are not relevant in the standalone mode, which would be activate depending on the `isStandalone` variable. Something like

```html
<div class="hide-if-standalone">

</div>

<script>
if (isStandalone) {
    for (const element of document.getElementsByClassName("hide-if-standalone") {
        element.style.display = "none";
    }
}
</script>
```

Comment (ceriottm) [reply app/index.html:211]: this would be great. you know how to implement it easily?

File: app/index.html (hunk 1 above)
Comment (ceriottm) [reply app/index.html:215]: I was thinking that this "static standalone viewer" can be used as a local GUI, and you have to load your dataset in it.

File: app/index.html (hunk 2 above)
Comment (Luthaf) [reply app/index.html:211]: The code above should work. Do you want to add this to this PR or leave it for later?

**Comment by ceriottm** [comment]
Ready to merge IMO @Luthaf

**Comment by Luthaf** [comment]
The commit history is very strange ... Did you `git rebase master` and then `git pull` `git push`? The same commits exist multiple times, and I see already on master. Usually, after a rebase, you want to do a force push with `gith push --force-with-lease` (--force-with-lease only force push if no one else pushed to the branch in the meantime)

**Comment by ceriottm** [comment]
uhm. might have done that, indeed. can be fixed or we need to nuke the
branch?

**Pull request merged**